qiita_db/study.py

```diff
@@ -340,6 +337,17 @@ def create(cls, owner, title, efo, info, investigation=None):
                "efo_id) VALUES (%s, %s)".format(cls._table))
         conn_handler.executemany(sql, [(study_id, e) for e in efo])
 
+        # insert portal into database
+        portal_id = convert_to_id(qiita_config.portal, 'portal_type', 'portal')
```

Comment:
Please, replace this block of code with a variation of the code that I wrote for the analysis object, as it is conceptually doing the same thing. It will also remove magic numbers and will be consistent.

Also the comment above is incorrect, as it is not adding any portal.